backend/src/test/java/heartbeat/service/report/KanbanServiceTest.java

```diff
@@ -33,8 +33,11 @@ class KanbanServiceTest {
 
 	@Test
 	void shouldCallCsvServiceToGenerateScvInfo() {
+		JiraBoardSetting mockJiraBoardSetting = KanbanFixture.MOCK_JIRA_BOARD_SETTING();
```

Comment:
for the test description `shouldCallCsvServiceToGenerateScvInfo` -> `shouldCallCsvServiceToGenerateCSVInfo`, yes?

Reply:
what is `when` for this test case

Reply:
done